/RealSenseDriver:
  ros__parameters:
    depth_wight: 1280
    depth_hight: 720
    depmin: 0.2
    depmax: 2.
    EXPOSURE: 166.
    GAIN: 10.
    BRIGHTNESS: 1.
